### ISSUE TITLE
List of NSEs

### ISSUE THREAD

**Issue by starblast** [ukncsc/SME#3]
Are you planning on offering/linking to/distributing a list of the NSEs somewhere or is the link only going in your blog?

**Comment by starblast** [comment]
Actually, it seems like you should be maintaining the NSEs in your own git repo. The point of this service (as i understand it) is that you verify that the NSE works and is not malicious. If you just distribute links, the owner of the link could change the NSE into something malicious after the fact. You need a point-in-time snapshot of the code that your team has vetted.

**Comment by CliffordNetwork** [comment]
https://github.com/nccgroup/nmap-nse-vulnerability-scripts

**Comment by starblast** [comment]
That's a link to an external team (nccgroup)'s NSEs. They can change the contents of that link at any time, so I'm not sure what the point of SME is in this context?

IMO a service like this should seek to compile vetted NSEs into a single location so someone could check out ONE git repo and then use those NSEs in their scanning.

**Comment by AkikoOrenji** [comment]
Agreed this is very confusing. Great initative but without centrally storing or managing these scripts whats the point.
